mariadb-10.6.yaml

```diff
@@ -1,6 +1,5 @@
 package:
-  name: mariadb
-  # Latest LTS
+  name: mariadb-10.6
```

Comment:
Do we want this to also provide `mariadb=10.6.999`? (and throughout for the subpackages)

Reply:
Fixed!